site/en/docs/webstore/publish/index.md

```diff
@@ -121,11 +121,29 @@ you uncheck the checkbox, your item will **not** be published immediately after
 complete. Instead, you'll be able to manually publish it at a time of your choosing once the review
 is complete.
 
+{% Aside %}
```

Comment:
Is there a reason we split the instructions between two sections? What if we removed the aside in the "Deferred publishing option" section and just add the instructions there rather than having them in the "Review of submitted items" section?

Reply:
The idea was that the "deferred publish option" section is happening before you click "submit for review". And going to the pending review to click on "Defer publish" can only happen once the review is pending.

That was my rationale; it also makes "oh, and if you want to change your mind" appear in an aside rather than as instructions inside the how to submit process -- these instructions are pushed out until later in case you need them then.

Thoughts on this?

Reply:
Ok, I don't feel strongly so I'm fine with this organization of content as well.